Integrations/integration-DNSDB.yml

```diff
@@ -0,0 +1,248 @@
+  required: true
+script:
+  script: |
+    var serverUrl = params.url;
+    if (serverUrl[serverUrl.length - 1] !== '/') {
+        serverUrl += '/';
+    }
+
+    var dict = {
+        rdata: {
+            url: 'lookup/rdata/',
+            defaultLimit:100,
+            tableColumns: {
+                rrname:'',
+                rrtype:'',
+                rdata:'',
+                count:'',
+                time_first:'ts',
+                time_last:'ts',
+                zone_time_first:'ts',
+                zone_time_last:'ts'
+            }
+        },
+        rrset: {
+            url: 'lookup/rrset/name/',
+            defaultLimit:100,
+            tableColumns: {
+                rrname:'',
+                rrtype:'',
+                rdata:'',
+                bailiwick:'',
+                count:'',
+                time_first:'ts',
+                time_last:'ts',
+                zone_time_first:'ts',
+                zone_time_last:'ts'
+            }
+
+        }
+    }
+    var valueExist = function(data,key) {
+        var array = (Array.isArray(data)) ? data : [data];
+
+        for (var i=0;i<array.length;i++) {
+            if (array[i][key]) {
+                return true;
+            }
+        }
+        return false;
+
+    }
+
+    var tsToTime = function(ts) {
+        if (ts) {
+            var date = new Date(ts*1000);
+            var hours = date.getHours();
+            var minutes = "0" + date.getMinutes();
+            var seconds = "0" + date.getSeconds();
+            var formattedTime = date + ' ' + hours + ':' + minutes.substr(-2) + ':' + seconds.substr(-2);
+            return formattedTime;
+        } else {
+            return '';
+        }
+    }
+
+    var arrToNewLines = function (data) {
+        var md='';
+        if (Array.isArray(data)) {
+            for (var i=0;i<data.length;i++) {
+                md += data[i] + '<br>';
+            }
+
+        } else {
+            md=data;
+        }
+        return md;
+    }
+
+    var dataToMd = function (api,data) {
+        var keys={};
+        for (key in dict[api].tableColumns) {
+            if (valueExist(data,key)) {
+                keys[key]=dict[api].tableColumns[key];
+            }
+        }
+        var head='|';
+        var line='-';
+        for (key in keys) {
+            head+=key+'|';
+            line+='-|';
+        }
+        var md=head+'\n'+line+'\n';
+        for (var i = 0; i<data.length; i++) {
+            md += '|';
+            for (key in keys) {
+                if (keys[key] == 'ts') {
+                    md += tsToTime(data[i][key])+'|';
+                } else {
+                    md += arrToNewLines(data[i][key]) + '|';
+                }
+            }
+            md += '\n';
+        }
+        return md;
+    }
+
+    var sendRequest = function(requestUrl,parameters) {
+        var url = serverUrl + requestUrl + encodeToURLQuery(parameters);
+        console.log(url);
+        var res = http(
```

Comment:
We're trying to make it a standard to at least add the proxy parameter to all integrations